ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -354,6 +356,13 @@
     ),
     reason="Test runs ONLY on Fusion HCI provider and client clusters",
 )
+dr_setup_required = pytest.mark.skipif(
+    not (
+        config.default_cluster_ctx.MULTICLUSTER.get("multicluster_mode")
```

Comment:
```suggestion
        config.MULTICLUSTER.get("multicluster_mode")
```
this should be enough